integration/common/openlineage/common/provider/dbt.py

```diff
@@ -601,6 +612,11 @@ def extract_namespace(self, profile: Dict) -> str:
                 f"Passed {profile['type']}"
             )
 
+    def dbt_version_facet(self):
+        return DbtVersionRunFacet(
+            dbtVersion=self.run_metadata['dbt_version']
```

Comment:
Is this nullable? It would seem so from line 141

Reply:
It's not. I've fixed line 141.